sentry-ruby/spec/spec_helper.rb

```diff
@@ -69,6 +70,12 @@
     skip("Skipping because one or more guards `#{guards.inspect}` returned false") if skip_examples
   end
 
+  config.around(:each, webmock: false) do |example|
+    WebMock.disable!
+    example.run
+    WebMock.enable!
+  end
```

Comment:
Should this live in `sentry-ruby/spec/contexts/with_request_mock.rb` too?

Reply:
@st0012 no because this context is cherry-picked only by a handful of specs and we want to be able to easily toggle webmock on or off on a per-spec basis.

I do want to say that the whole setup could be done nicer, but it's a nice to have at this point I think :) I can get back to it on another occasion.

Reply:
Yeah that's not a problem I'm just curious. And sorry that it's mostly the mess I left 😬 
I'll take a look at the setup this week as well. Thanks for fixing it.